### ISSUE TITLE
Wrong json output format for "oci audit event list" with streaming enabled if output is null.

### ISSUE THREAD

**Issue by dbarj** [oracle/oci-cli#204]
I have a code that will loop over all regions and compartments and create the json output for audit events. The problem is that some compartments/regions have nothing to return.

$ oci -v
2.6.6

When I run it without the streaming:
```
$ oci --cli-rc-file /dev/null --profile AEG --region ap-mumbai-1 audit event list --all --start-time 2019-08-30T00:00:00Z --end-time 2019-08-31T00:00:00Z --compartment-id ocid1.compartment.oc1..xxx
$
```

With streaming:
```
$ oci --cli-rc-file /dev/null --profile AEG --region ap-mumbai-1 audit event list --all --stream-output --start-time 2019-08-30T00:00:00Z --end-time 2019-08-31T00:00:00Z --compartment-id ocid1.compartment.oc1..xxx
{ "data":
}
$
```

The json output for streaming is invalid. It should be:
```
{ "data": null
}
```

And if for some reason my privileges are revoked, the error become even worse with the streaming option:
 
```
$ oci --cli-rc-file /dev/null --profile AEG --region ap-mumbai-1 audit event list --all --start-time 2019-08-30T00:00:00Z --end-time 2019-08-31T00:00:00Z --compartment-id ocid1.compartment.oc1..xxx
ServiceError:
{
    "code": "NotAuthenticated",
    "message": "The required information to complete authentication was not provided or was incorrect.",
    "opc-request-id": "753FB4F7394040FCB4C217412AAE1B89/68DC4408B1705ABE1F86EE9319E59231/278C089074991D38FD7C351BCD19872F",
    "status": 401
}
$
$ oci --cli-rc-file /dev/null --profile AEG --region ap-mumbai-1 audit event list --all --stream-output --start-time 2019-08-30T00:00:00Z --end-time 2019-08-31T00:00:00Z --compartment-id ocid1.compartment.oc1..xxx
{ "data":
}
AttributeError: 'NoneType' object has no attribute 'status'
$
```
Thanks,
Rodrigo Jorge

**Comment by paul-hummel-oracle** [comment]
@dbarj thank you for reporting this issue. We will investigate and post updates here.

**Comment by paul-hummel-oracle** [comment]
@dbarj this was fixed with v2.6.9.
